LinkBase/LinkBase/HudView.swift

```diff
@@ -7,3 +7,64 @@
 //
 
 import Foundation
+import UIKit
+
+class HudView: UIView{
+    var text = ""
+    
+    class func hud(inView view: UIView, animated: Bool) -> HudView {
+        
+        let hudView = HudView(frame: view.bounds)
+        hudView.isOpaque = false
+        
+        view.addSubview(hudView)
+        view.isUserInteractionEnabled = true
+        
+        hudView.show(animated: animated)
+        return hudView
+    }
+    
+    override func draw(_ rect: CGRect) {
+        
+        let boxWidth: CGFloat = 96
+        let boxHeight: CGFloat = 96
+        
+        let boxRect = CGRect(x: round((bounds.size.width - boxWidth) / 2), y: round((bounds.size.height - boxHeight)/2), width: boxWidth, height: boxHeight)
+        
+        let roundedRect = UIBezierPath(roundedRect: boxRect, cornerRadius: 10)
+        UIColor(white: 0.3, alpha: 0.8).setFill()
```

Comment:
As mentioned above, it's not implemented with auto layout at this stage, but rather displays as fixed size in center of screen (displays after pressing submit on the CommsVC)